u_public.go

```diff
@@ -168,7 +240,9 @@ func (chs12 *clientHandshakeState) toPublic12() *PubClientHandshakeState {
 // }
 
 type CertificateRequestMsgTLS13 struct {
-	Raw                              []byte
+	// Deprecated: crypto/tls no longer use this variable
```

Comment:
Should we delete it? It looks like we still use it, does it have no effect?

Reply:
Added some more docs on how it is not used. It's only not deleted for backwards compatibility, but I doubt anyone is actually using it so it could go either way.